web/src/components/FormStateStart.js

```diff
@@ -23,42 +23,33 @@ const FormStateStart = ({ handleSubmit, pristine, reset, submitting }) => (
 
     <SectionHeader>Your state:</SectionHeader>
     <Field
-      name="stateName"
+      name="state"
       component={SelectInput}
       label="State"
-      options={STATES}
+      options={[{ id: '', name: '--' }, ...STATES]}
     />
 
-    {false && (
-      <div>
-        <button type="submit" disabled={pristine || submitting}>
-          Submit
-        </button>
-        <button type="button" disabled={pristine || submitting} onClick={reset}>
-          Clear Values
-        </button>
-      </div>
-    )}
+    <div className="mt3">
+      <button
+        type="submit"
+        className="btn btn-primary bg-green"
+        disabled={submitting}
+      >
+        {submitting ? 'Saving' : 'Submit'}
+      </button>
+    </div>
   </form>
 );
 
 FormStateStart.propTypes = {
   handleSubmit: PropTypes.func.isRequired,
-  pristine: PropTypes.bool.isRequired,
-  reset: PropTypes.func.isRequired,
   submitting: PropTypes.bool.isRequired
```

Comment:
❗️ Where does `submitting` come from?  Should it be wired up from the StateStart container?

Reply:
auto-generated by `reduxForm` -- https://redux-form.com/7.2.3/docs/api/props.md/#-code-props-code-

Reply:
Oh, I should have searched harder.  That's really handy!